prophet/handlers/pricepred/pricepred.go

```diff
@@ -97,7 +97,7 @@ func (s PricePredictorSolidity) Execute(ctx context.Context, input []byte) ([]by
 
 	var backtestingRes *BacktestingResponse
 	if len(inputData.Metrics) > 0 {
-		res, err := s.c.Backtesting(ctx, BacktestingRequest{
+		res, err := s.c.Backtesting(ctx, BacktestingRequest{ //nolint:all
```

Comment:
`S1016: should convert req (type PredictRequest) to BacktestingRequest instead of using struct literal (gosimple)`

Can you please make method that will convert PredictRequest to BacktestingRequest?

Reply:
+